docs/index.rst

```diff
@@ -1,20 +1,19 @@
 :mod:`pyramid_nacl_session`
 ===========================
 
-This Pyramid add-on defines an encrypting, pickle-based cookie serializer,
+This Pyramid add-on defines an encrypting, pickle-based cookie serializer, 
 using ``PyNaCl`` [1] to generate the symmetric encryption for the cookie state.
 
-
 [1] https://pynacl.readthedocs.io/en/latest/secret/
 
 Contents:
 
 .. toctree::
-   :maxdepth: 2
+    :maxdepth: 2
```

Comment:
I didn't notice this before, but the 3 spaces vs the 4 is intentional. In restructured text documents you match the `.. ` spacing not the usual 4 spaces in Python code.

Reply:
We want 4 spaces for consistency. Editors will incorrectly indent Python code in `code-block` because they interpret the first indent as the file's default.

Reply:
Oh, okay.